### DISCUSSION TITLE
Enum Order

### DISCUSSION THREAD

**Discussion by onursimsek** [filamentphp/filament#12043] [Ideas]
Hi there,

We can use label (HasLabel) and description (HasDescription) with Enums, that's great but we can't sort Enums.

Can we add HasOrder interface for Enums? If you want i can send a PR @danharrin

**Comment by danharrin** [comment]
The order is the order in which cases are defined in the file, is that not enough?

**Reply by gundogduyakici** [reply]
Why do I have to use selectbox in the same order every time I use it? Custom sorting is needed for different status priorities to the customer on different pages. Don't limit us 👎🏻 Yes, not enough.

**Reply by danharrin** [reply]
> Why do I have to use selectbox in the same order every time I use it? Custom sorting is needed for different status priorities to the customer on different pages. Don't limit us 👎🏻 Yes, not enough.

😂 Just define the `options()` then in your order, if they are in a different order on different pages. The original thread was about changing the order for all enum usages throughout the app
